docs/topics/ref_known-issues-7-1-1.adoc

```diff
@@ -0,0 +1,17 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-14
+
+:_mod-docs-content-type: REFERENCE
+
+[id="known-issues-7-1-1_{context}"]
+= Known issues
+
+This section provides highlighted known issues in {ProductFullName} version 7.1.1.
+
+.Unable to perform bulk analysis on Windows systems
+
+In {ProductShortName} CLI 7.1.0, bulk analysis on Windows systems cannot be performed. Currently, there is no workaround available. (link:https://issues.redhat.com/browse/MTA-3624[MTA-3624])
+
+.Binary analysis fails with incorrect Maven credentials
+
+Binary analysis fails with a generic error when incorrect Maven credentials are used. This occurs because the analyzer is unable to report detailed error messages. (link:https://issues.redhat.com/browse/MTA-3714[MTA-3714])
```

Comment:
```suggestion
Binary analysis fails with a generic error when incorrect Maven credentials are used. The failure occurs because the analyzer is unable to report detailed error messages. (link:https://issues.redhat.com/browse/MTA-3714[MTA-3714])
```